src/adv.py

```diff
@@ -36,16 +36,31 @@
 #
 # Main
 #
-
 # Make a new player object that is currently in the 'outside' room.
-
+#
 # Write a loop that:
 #
 # * Prints the current room name
 # * Prints the current description (the textwrap module might be useful here).
 # * Waits for user input and decides what to do.
 #
 # If the user enters a cardinal direction, attempt to move to the room there.
+#
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+Player1 = Player(name=input("Please choose player name: "), desc=input("Enter Player Class: "), satchel=" no items", location=room['outside'])
+
+playerDesc = []
+
```

Comment:
Awesome work Javier, you can certainly add more to this such as the ability for you player to get points!